src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -80,8 +80,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(DeployJob.class);
     private static final String bundlePrefix = "bundles";
     public static final String DEFAULT_INSTANCE_TYPE = "t2.medium";
-    private static final String AMI_CONFIG_PATH = "modules.deployment.ec2.default_ami";
-    private static final String DEFAULT_AMI_ID = DataManager.getConfigPropertyAsText(AMI_CONFIG_PATH);
+    public static final String AMI_CONFIG_PATH = "modules.deployment.ec2.default_ami";
```

Comment:
This constant is now only used in this file for a log statement. This makes me think that it should probably be moved to EC2Info?